src/applications/hca/validation.js

```diff
@@ -79,10 +77,11 @@ export function validateDependentDate(
   messages,
   index,
 ) {
-  const dependent = moment(dependentDate);
-  const dob = moment(_.get(`dependents[${index}].dateOfBirth`, formData));
+  const dependent = Date.parse(dependentDate);
+  const dob =
+    Date.parse(formData?.dependents?.[index]?.dateOfBirth) || Date.now();
```

Comment:
This is a bit confusing. The arguments in the previous `_.get()` call seem swapped - [the object should be first and the path second](https://lodash.com/docs/4.17.15#get).

However, the `_.get()` call was _likely_ resulting in `undefined` each time, and [`moment(undefined)` returns the current datetime](https://momentjs.com/docs/#/parsing/now/):

> Note: Function parameters default to undefined when not passed in. Moment treats moment(undefined) as moment().

Reply:
if the optional chaining returns `undefined`, `Date.parse(undefined)` will be `NaN`, so the `|| Date.now()` should achieve the same thing that was previously happening.

Reply:
>The arguments in the previous _.get() call seem swapped

It's [lodash/fp](https://gist.github.com/jfmengels/6b973b69c491375117dc#_getpath-object).

> if the optional chaining returns undefined, Date.parse(undefined) will be NaN, so the || Date.now() should achieve the same thing that was previously happening

This looks fine. maybe add a comment explaining that `Date.parse` returns `NaN` when applied to `undefined`.